configs/CSC-LUMI-HPECray-zen3-MI250X-Slingshot/software.yaml

```diff
@@ -3,23 +3,24 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-spack:
+software:
   packages:
     default-compiler:
-      spack_spec: cce@16
+      pkg_spec: cce@16
     default-mpi:
-      spack_spec: cray-mpich@8.1%cce ~gtl
+      pkg_spec: cray-mpich@8.1%cce ~gtl
     compiler-rocm:
-      spack_spec: cce@16
+      pkg_spec: cce@16
     blas-rocm:
-      spack_spec: rocblas@5.5
+      pkg_spec: rocblas@5.6.1
     blas:
-      spack_spec: rocblas@5.5
+      pkg_spec: cray-libsci@23
```

Comment:
Why is this changing? It seems unrelated to the PR

Reply:
Yes, that is unrelated to the PR. I don't know where these changes came from. Same with the version on blas-rocm.